src/views/ConsentAmalgamationOut.vue

```diff
@@ -820,7 +831,7 @@ export default class ConsentAmalgamationOut extends Mixins(CommonMixin, DateMixi
 
     const data: any = {
       [FilingTypes.CONSENT_AMALGAMATION_OUT]: {
-        details: `${this.defaultComment}\n${this.detailComment}`,
+        details: this.isRoleStaff ? `${this.defaultComment}\n${this.detailComment}` : null,
```

Comment:
This is where the filing is saved.

The code to fetch the filing (on resume) works fine as-is.

Reply:
This is outdated code -- now we don't save the `details` at all.